scopes/lanes/lanes/lane.cmd.ts

```diff
@@ -336,7 +336,8 @@ export class LaneMergeCmd implements Command {
 }
 
 export class LaneRemoveCmd implements Command {
-  name = 'remove <lane...>';
+  name = 'remove <lanes...>';
+  arguments: [{ name: 'lanes...'; description: 'A list of lane names, separated by spaces' }];
```

Comment:
when an arg has the array form: `...`, in the CLI it is separated by a space. not a comma.

Reply:
not sure what you mean (i wrote 'space', not 'comma')

Reply:
oops.. my bad